demo/pages/index.js

```diff
@@ -1,13 +1,31 @@
-import React from 'react';
-import NewComponent from '@moxy/react-lib-template';
+import React, { useState, useCallback } from 'react';
+import Circle from '@moxy/react-circle';
 
 import styles from './index.module.css';
 
-const Home = () => (
-    <div className={ styles.home }>
-        <h1>react-lib-template</h1>
-        <NewComponent className={ styles.newComponent }>Hello World</NewComponent>
-    </div>
-);
+const Home = () => {
+    const [percentage, setPercentage] = useState(0.3);
+    const [counter, setCounter] = useState(0);
+    const buttonCallback = useCallback(() => {
+        setPercentage(percentage === 0.3 ? 0.7 : 0.3);
```

Comment:
I think a demo ending with the full circle would work better for people to understand what this can do.

Reply:
Honestly I'm not sure which way I prefer, I think it's indifferent. Would you prefer for it to start at `0.3` and end at `1`? I think it's important for one of them not to be an integer to show you can have a partial stroke.

Reply:
Agree, so starting at `0.3` and finishing at `1` would work!